examples/test/Export_test.js

```diff
@@ -0,0 +1,71 @@
+// (C) 2007-2020 GoodData Corporation
+import { Selector } from "testcafe";
+import fs from "fs";
+import { config } from "./utils/config";
+import { loginUsingLoginForm } from "./utils/helpers";
+import { enableDownloadForHeadlessChrome, exportToCSV, exportToExcel } from "./utils/exportUtils";
+
+const exportButtonsCSS = ".gd-button-secondary";
+const getExportCSVButton = chartSelector => {
+    return chartSelector
+        .parent()
+        .find(exportButtonsCSS)
+        .withText("Export CSV");
+};
+const getExportExcelButton = chartSelector => {
+    return chartSelector
+        .parent()
+        .find(exportButtonsCSS)
+        .withText("Export XLSX");
+};
+const getExportWithCustomNameButton = chartSelector => {
+    return chartSelector
+        .parent()
+        .find(exportButtonsCSS)
+        .withText("Export with custom name CustomName");
+};
+
+fixture("Export")
+    .page(config.url)
+    .beforeEach(loginUsingLoginForm(`${config.url}/export`));
+
+test("Export chart data", async t => {
+    const barChart = Selector(".s-bar-chart");
+    const fileName = "BarChart";
+    await enableDownloadForHeadlessChrome(t);
+    await t
+        .click(await getExportExcelButton(barChart))
```

Comment:
not sure, if inside of click action you need to use await, see doc
https://devexpress.github.io/testcafe/documentation/test-api/actions/click.html